### PR TITLE
fix(scoop-info): Fix error message when manifest is not found

### LINKED ISSUE #4928: [Feature] (Cosmetic) if manifest not found, then scoop just reports the last-searched-bucket/manifest
## Feature Request

(Could be a bug, but is mainly cosmetic). If you try to do `scoop info` on a manifest that can't be found, then scoop reports that the manifest can't be found in the last bucket searched rather than just simply saying, "Couldn't find it in any bucket".

It's easiest to describe the behaviour with some logging : 

```powershell
PS C:\Users\Me> scoop bucket list

Name      Source                                    Updated             Manifests
----      ------                                    -------             ---------
extras    https://github.com/ScoopInstaller/Extras  16/05/2022 12:05:35      1501
java      https://github.com/ScoopInstaller/Java    14/05/2022 21:01:48       220
jetbrains https://github.com/Ash258/Scoop-JetBrains 13/05/2022 21:07:27        99
main      https://github.com/ScoopInstaller/Main    16/05/2022 10:43:16      1020
tilt-dev  https://github.com/tilt-dev/scoop-bucket  12/05/2022 22:24:02         0

PS C:\Users\Me scoop search posh
'extras' bucket:
    posh-docker (0.7.0)
    posh-git (1.1.0)

'main' bucket:
    oh-my-posh (7.85.0)

### This is is the problem command, mainly because I can't spell!
PS C:\Users\Me> scoop info pos-git
Could not find manifest for 'tilt-dev/pos-git'.
```

#### Describe the solution you'd like

It feels like it's as simple as changing https://github.com/ScoopInstaller/Scoop/blob/aaa726c09f786d2af902d827ae8d0caee964ca1e/libexec/scoop-info.ps1#L32-L34

so that you don't pass in `$bucket` so we can just have something like

```powershell
if (!$manifest) {
    abort "Could not find manifest for '$(show_app $app)' in any buckets"
}
```

```
PS C:\Users\Me> scoop info pos-git
Could not find manifest for 'pos-git' in any buckets.
```

### PR DESCRIPTION
**Pull request by quotidian-ennui**
#### Description

Fixes https://github.com/ScoopInstaller/Scoop/issues/4928.

Abort message no longer references the last bucket searched via `Find-Manifest`

#### Motivation and Context

Fixes #4928

#### How Has This Been Tested?

This is an edge case abort message, and there appear to be no tests that use the string literal `Could not find manifest` which suggests the text is never explicitly matched against during test execution. 

```
$ find . -name "*.ps1" -exec grep -iH "could not find manifest" {} \;
./libexec/scoop-home.ps1:        abort "Could not find manifest for '$app'."
./libexec/scoop-info.ps1:    abort "Could not find manifest for '$(show_app $app)' in any known buckets."
```

My assumption here is that the tests should pass, sadly I don't actually have a powershell build environment, and I wouldn't really know how to start building one to include `BuildHelpers` and other PS modules referenced in _test/bin/test.ps1_

#### Checklist:

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I have ensured that I am targeting the `develop` branch.
- [ ] ~I have updated the documentation accordingly.~ Documentation change not required.
- [ ] I have updated the tests accordingly.
- [x] I have added an entry in the CHANGELOG.

### CODE REVIEW

File: libexec/scoop-info.ps1 [hunk 1]
```diff
@@ -30,7 +30,7 @@ if ($app -match '^(ht|f)tps?://|\\\\') {
 }
 
 if (!$manifest) {
-    abort "Could not find manifest for '$(show_app $app $bucket)'."
+    abort "Could not find manifest for '$(show_app $app)' in any buckets."
```
Comment (HUMORCE) [inline libexec/scoop-info.ps1:33]:
```suggestion
    abort "Could not find manifest for '$(show_app $app)' in local buckets."
```

**Review by HUMORCE — APPROVED** [review]

**Review by niheaven — DISMISSED** [review]
